pkg/transcriptionsystems/bipa/consonants.tsv

```diff
@@ -439,6 +441,7 @@ d̠ʒ	voiced	post-alveolar	affricate	+	sibilancy:sibilant
 dʒ	voiced	post-alveolar	affricate		sibilancy:sibilant	
 dːʒ	voiced	post-alveolar	affricate		duration:long,sibilancy:sibilant	
 dʒʲ	voiced	post-alveolar	affricate		palatalization:palatalized,sibilancy:sibilant	
+d̠ʓ	voiced	post-alveolar	affricate	+	palatalization:palatalized,sibilancy:sibilant	
```

Comment:
Again, an example for an abuse of an alias, as this is the notation they use in phoible, but we do not need to use it. I'd rather propose that those dataset which use the form will be explicitly linked to our value.

Reply:
Even more so, I doubt if this is post-alveolar, isn't the stroke under it a marker for a dental?

Reply:
I will move it to Phoible as well.